test/spectral/tests/disabled-rules.test.js

```diff
@@ -0,0 +1,659 @@
+const commandLineValidator = require('../../../src/cli-validator/runValidator');
+const { getCapturedText } = require('../../test-utils');
+
+describe('spectral - test disabled rules - Swagger 2', function() {
+  let consoleSpy;
+
+  beforeEach(() => {
+    consoleSpy = jest.spyOn(console, 'log').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    consoleSpy.mockRestore();
+  });
+
+  it('test contact-properties rule using mockFiles/swagger/disabled-rules.yml', async function() {
+    // set up mock user input
+    const program = {};
+    program.args = ['./test/spectral/mockFiles/swagger/disabled-rules.yml'];
+    program.default_mode = true;
+
+    const exitCode = await commandLineValidator(program);
+    const capturedText = getCapturedText(consoleSpy.mock.calls);
+    const allOutput = capturedText.join('');
+
+    expect(exitCode).toEqual(1);
```

Comment:
If no rules are triggered, the exit code should be 0

Reply:
Well this whole test suite tests that Spectral rules are disabled, so regular validator rules can still be triggered. In this test case, some non-spectral rules triggered errors in validation. I thought of creating a separate api file for each spectral rule I was testing, to make sure other validations weren't being triggered but I figured that would be overkill.

Reply:
That's reasonable - will you just add a comment explaining that? It was confusing to see a "positive test" with an exit code of 0